currency/aggregate_conversions_test.go

```diff
@@ -0,0 +1,79 @@
+package currency
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGroupedGetRate(t *testing.T) {
+
+	// Setup:
+	customRates := NewRates(time.Now(), map[string]map[string]float64{
+		"USD": {
+			"GBP": 0.77208,
+			"EUR": 0.80,
+		},
+	})
+
+	pbsRates := NewRates(time.Now(), map[string]map[string]float64{
+		"USD": {
+			"GBP": 0.50,
+			"MXN": 10.31,
+		},
+	})
```

Comment:
I'd also suggest having a test case where a reverse currency conversion is present as an override in the custom. currency rates. So for example, here maybe you can add `MXN` to `USD` in custom rates and make sure that works as expected as well?

Reply:
I believe we already have that covered in the test cases. Did you have any othre scenario un mind? 
```
36     testGroups := []struct {
37         expectError bool
38         testCases   []aTest
39     }{
40         {
41             expectError: false,
42             testCases: []aTest{
43                 {"Found in both, return custom rate", "USD", "GBP", 0.77208},
44 /*OVERRIDE-->*/ {"Found in both, return inverse custom rate", "GBP", "USD", 1 / 0.77208},
45                 {"Found in custom rates only", "USD", "EUR", 0.80},
46                 {"Found in PBS rates only", "USD", "MXN", 10.31},
47                 {"Found in PBS rates only, return inverse", "MXN", "USD", 1 / 10.31},
48                 {"Same currency, return unitary rate", "USD", "USD", 1},
49             },
50         },
currency/aggregate_conversions_test.go
```